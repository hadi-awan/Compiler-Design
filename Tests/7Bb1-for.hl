// for loops on a list of expressions
var i;
for i in 1 do print i, " "; end; println;
for i in 1,3,46,8 do print i, " "; end; println;
for i in {1,3,46,8} do print i, " "; end; println;
for i in -1,{-2,-3,-4},{-5,-6},-10 do print i, " "; end; println;