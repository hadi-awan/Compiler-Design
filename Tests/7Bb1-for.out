1 
1 3 46 8 
1 3 8 46 
-1 -4 -3 -2 -6 -5 -10 
